pages/common/neofetch.md

```diff
@@ -0,0 +1,24 @@
+# neofetch
+
+> CLI system tool to display information about your operating system, software and hardware.
+
+- Call neofetch to return default config. Config file will be created on first run at $HOME/.config/neofetch/config.conf:  
+
+`neofetch`
+
+- Pass specific parameters to alter what gets displayed:
+
+`neofetch --{{opt1}} {{value}} --{{opt2}} {{value}}`
+
+- Enable/disable an info line from appearing in the output:
+> NOTE: 'infoname' is the function name from the 'print_info()' function inside the config file.
```

Comment:
Notes aren't supported in tldr syntax. :smile:

Reply:
Sorry about that. And thanks for helping me through all the details!

Reply:
No need to apologise. :+1: Thanks for contributing.